wfm

```diff
@@ -249,6 +249,37 @@ mongo () {
   #
 }
 
+redis () {
+  PID=`ps x | grep -v grep | grep "redis-server" | awk '{ print $1 }'`
+  case $1 in
+    start)
+    if [ ! -z $PID ]; then
+      echo "Redis is already running. pid: $PID"
+    else
+      redis-server &
```

Comment:
here I'd check if redis is installed or at the beginning of funtion. If thats not then exit and show some message